joomla/joomla-cms
# Add access checks to the plugin parameter

At the moment every user gets the same tinyMCE button structure, regardless of his/her permissions/access level.
With this patch, all buttons from the complete mode can be linked to an access level, so you can customise the editor by access level.

Test:
1. Set the editor mode to "complete"
2. Apply patch => editor should work as before (so no B/C issues)
3. Look into the editor parameters, here you can now define access levels on the second tab
4. Play with the different combinations and see how the editor buttons change
